tests/test_matplotlib.py

```diff
@@ -12,6 +13,18 @@ def test_plot(selenium_standalone_matplotlib):
     plt.show()
 
 
+@pytest.mark.skip(reason="wrong version of matplotlib_pyodide in tests")
```

Comment:
Is it not testing the current code base? We should fix that (not in this MR)

Reply:
Yes, this is my understanding. I've had some issues running these tests, which depend on pytest-pyodide. On brief inspection, near the error I noticed `dist_dir` was set to `pyodide` not `matplotlib_pyodide`, so there might be an issue with `pytest-pyodide`. Its not an `env` issue like I first thought because I tried to run the suite on the pyodide `./run_docker` image too, which resulted in the same error. However, it needs more thorough introspection and is not in scope here.